### ISSUE TITLE
Help in reproducing results on coco

### ISSUE THREAD

**Issue by ayushjain1144** [fundamentalvision/Deformable-DETR#77]
Hi, 

Thanks for your awesome work. I am trying to train ddetr on coco by running `GPUS_PER_NODE=4 ./tools/run_dist_launch.sh 4 ./configs/r50_deformable_detr.sh`. I am getting the following results:

```
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.374
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.559
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.403
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.218
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.405
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.493
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.318
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.530
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.572
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.343
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.620
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.766
```

The only difference is of batch size: I am using batch size of 4 with 4 GPUs, so a total batch size of 16. Could you suggest some things that I should try to get similar performance as yours? Do I need to train for more epochs or use a different learning rate? Any insights would be helpful! Thanks!

**Comment by IgorDroz** [comment]
@ayushjain1144 Hey, could you please share your list of packages(i mean the 'pip freeze' output)? I am just unable to reproduce the results and i wonder if the packages version is my problem
